helper/experiments/experiments.go

```diff
@@ -4,9 +4,12 @@
 package experiments
 
 const (
-	VaultExperimentEventsAlpha1          = "events.alpha1"
 	VaultExperimentCoreAuditEventsAlpha1 = "core.audit.events.alpha1"
 	VaultExperimentSecretsSyncAlpha1     = "secrets.sync.alpha1"
+
+	// Unused experiments. We keep them so that we don't break users who include them in their
```

Comment:
I agree we should keep the experiment definition but I think we should probably generate a nice error message instead of silently ignoring - it's surprising for users if they try to enable an experiment that they think does something but it doesn't.